policies/access.md

```diff
@@ -0,0 +1,35 @@
+# GitHub Access Management Policy
+
+## Teams, not Individuals
```

Comment:
One commonly encountered downfall of using GitHub teams is that the teams and their members are not publicly visible. Teams have two states, [visible and secret](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#team-visibility). Visible teams can be viewed and mentioned **by organisation members**. Secret teams can be viewed and mentioned only by people on the team and organisation owners. 

This observation need not be a blocker for this effort, but we should likely document it here and be more aware of it in how we communicate governance. For example, [linking to teams in README's](https://github.com/ossf/wg-securing-software-repos/issues/15) isn't very useful for the majority of folks reading the README.

Reply:
I may be wrong here, I can see 9 of the 44 teams in this view https://github.com/orgs/ossf/teams which suggests there's a third visibility state for teams.

Reply:
I'm pretty sure I don't see any teams, just repositories. Try browsing that page using incognito?

Reply:
I’m aware of this, yes - the long term intention is to automate things so that the readme will list the members of a team, and stay in sync with it. If folks think it’s worth documenting how github works here that’s fine, but I’m not going to hold up this PR on it.

Reply:
Since one of your reviewers tripped on this behavior, and I doubt most GitHub users are well versed in the details, and it sounds like you have quite a few members, it's probably a good idea to add some text to mention all of these points.

Reply:
oppose re-documenting GH here, or anywhere.  at most, linking to [the relevant documentation](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#team-visibility) is sufficient